### PR TITLE
scroll: new package at 0.1

### PR DESCRIPTION
**Pull request by mcpcpc**
Please fill out the template.

Read: https://k1ss.org/style
Read: https://k1ss.org/guidestones

---

## Description of package

 This program provides a scroll back buffer for a terminal like st.

## Installed manifest of package

(`cat /var/db/kiss/installed/<pkg_name>/manifest`)

```

/var/db/kiss/installed/scroll/version
/var/db/kiss/installed/scroll/sources
/var/db/kiss/installed/scroll/manifest
/var/db/kiss/installed/scroll/checksums
/var/db/kiss/installed/scroll/build
/var/db/kiss/installed/scroll/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/scroll.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/bin/scroll
/usr/bin/
/usr/

```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/scroll/sources [hunk 1]
```diff
@@ -0,0 +1 @@
+git+git://git.suckless.org/scroll
```
Comment (dylanaraps) [inline community/scroll/sources:1]: Any reason for using a git source?

Comment (mcpcpc) [reply community/scroll/sources:1]: As far as i could tell, suckless.org does not maintain a release tarball on their site for this program. although, im still a newb and may have missed something obvious :). 

src: https://dl.suckless.org

Comment (dylanaraps) [reply community/scroll/sources:1]: Alrighty.

File: community/scroll/version [hunk 2]
```diff
@@ -0,0 +1 @@
+0.1 1
```
Comment (dylanaraps) [inline community/scroll/version:1]: This should then be `git 1`.

Comment (mcpcpc) [reply community/scroll/version]: fixed

**Pull request merged**